drizzlepac/hapsequencer.py

```diff
@@ -442,6 +452,73 @@ def run_hap_processing(input_filename, diagnostic_mode=False, use_defaults_confi
 
 # ----------------------------------------------------------------------------------------------------------------------
 
+def run_sourcelist_comparision(total_list,log_level=logutil.logging.INFO):
+    """ This subroutine automates execution of drizzlepac/devutils/comparison_tools/compare_sourcelist_flagging.py to
```

Comment:
To be consistent with the rest of the code, the log_level should be a required parameter to be passed into the run_sourcelist_comparison and not have a default value for this routine.  The default value is set in runsinglehap.py.